zerver/lib/exceptions.py

```diff
@@ -656,3 +657,15 @@ def __init__(self) -> None:
     @override
     def msg_format() -> str:
         return _("You do not have permission to use topic wildcard mentions in this topic.")
+
+
+class PreviousSettingValueMismatchedError(JsonableError):
+    code: ErrorCode = ErrorCode.EXPECTATION_MISMATCH
+
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    @override
+    def msg_format() -> str:
+        return _("'old' value does not match the expected value.")
```

Comment:
Is this error fine for here considering we were discussing about showing different message in the client as we pass a unique error code?

Reply:
Have not changed the error message to be different in client.